### ISSUE TITLE
[Documentation] Add nullable fields in the API documentation

### ISSUE THREAD

**Issue by Tails128** [forem/forem#9061]
**Is your feature request related to a problem? Please describe.**
It would be super nice to have written in the documentation which response parameters can be null or absent in the answer, in order to write better typescript code.
[Edit] Some fields are already `null`, but some are not e.g: `organization` in published articles

**Describe alternatives you've considered**

I am currently writing my library so I'm currently going for some trial and error, but just because I always get a field doesn't mean it is not nullable.

**Comment by jacobherrington** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by nickytonline** [comment]
Is this something you'd like to tacke @Tails128 ? If so, let us know and we can assign the issue to you.

**Comment by Tails128** [comment]
truth be told I would like to tackle it, but I need to clarify something with my current job first, my contract requires written approval even for working on opensource stuff (I will refrain to comment about it...)

**Comment by jkrsn98** [comment]
Hey @jacobherrington @nickytonline, if you want this added to the documentation I'd like to work on it.

**Comment by nickytonline** [comment]
@jkrsn98, we're just waiting to hear back from @Tails128 to see if their work cleared them to work on this.

**Comment by Tails128** [comment]
@nickytonline sorry, I was awaiting for an answer, currently I cannot help you :s

**Comment by nickytonline** [comment]
All yours @jkrsn98!

**Comment by paulopacitti** [comment]
It has been already solved, this issue can be closed

**Comment by rhymes** [comment]
Thanks @paulopacitti!
